Automattic/wp-calypso
# Editor: Revisions: Diff revisions only once

Previously, revisions were diffed on receive time just to get the count of added and removed words, but the exact diff wasn't saved and had to be recomputed again from scratch once the revision was opened. This PR saves the whole diff to state, together with change summary. The counter now also reflects both changes in the content and in the title.

To test:
- open a post with revisions
- open revisions in the sidebar
- make sure you still see "words added, words removed" summaries in the revision list and it takes into count both title and content updates
- switching revisions should have no significant delay, as they are precomputed
